crates/re_renderer/src/mesh.rs

```diff
@@ -79,12 +84,87 @@ pub mod mesh_vertices {
 pub struct Mesh {
     pub label: DebugLabel,
 
+    /// Length must be a multipel of three.
     pub indices: Vec<u32>, // TODO(andreas): different index formats?
+
     pub vertex_positions: Vec<glam::Vec3>,
-    pub vertex_data: Vec<mesh_vertices::MeshVertexData>,
+
+    /// Per-vertex albedo color with unmulitplied/separate alpha.
+    /// Must be equal in length to [`Self::vertex_positions`].
+    pub vertex_colors: Vec<[u8; 4]>,
```

Comment:
I wanted to comment on why we're not using `Vec<re_renderer::Color32>` here, but that would have the semantics of pre-multiplied of course. Makes me wonder if we should have `re_renderer::Color32Unmultiplied` of whether that's overengineering 🤔

Reply:
I think that would be good. Explicit is better than implicit.